typespec/Applications.Core/containers.tsp

```diff
@@ -38,27 +38,27 @@ using OpenAPI;
 namespace Applications.Core;
 
 model ContainerResource is TrackedResource<ContainerProperties> {
-  @doc("Container name")
+  @doc("The name of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information.")
   @path
   @key("containerName")
   @segment("containers")
   name: ResourceNameString;
 }
 
-@doc("Container properties")
+@doc("Properties of the resource.")
```

Comment:
nit: referring to container as 'resource' in updated comment. we're not doing that in other comment. original comment is actually in line with updated comment on line 52. update may not be needed?